cadc-permissions/src/test/java/org/opencadc/permissions/TokenToolTest.java

```diff
@@ -140,15 +144,33 @@ public void testRoundTripToken() {
                 "C=CA, O=Grid, OU=nrc-cnrc.gc.ca, CN=Brian Major",
                 "C=CA, O=Grid, OU=nrc-cnrc.gc.ca, CN=Brian Major",};
 
+            // keys in file
             TokenTool gen = new TokenTool(pubKeyFile, privateKeyFile);
             TokenTool ver = new TokenTool(pubKeyFile);
+            // keys in memory
+            KeyPairGenerator kpg;
```

Comment:
this setup code (lines 151-160) would be replaced by call to `RsaSignatureGenerator.generateKeyPair(int)`.

Reply:
Yeah. I thought it might be too simple to bother with.